server.js

```diff
@@ -37,7 +37,8 @@ app.use(express.static('public'))
 app.post(['/workflows/*', '/triggers/*'], (req, res) => {
     res.send('ok');
     const payload = req.body;
-    const flatPayload = flatten(payload);
+    // Slack restricts variable keys to "a combination of letters, numbers, hyphens, and underscores."
+    const flatPayload = flatten(payload, { delimiter: '_' });
```

Comment:
Perhaps a query string parameter would be appropriate to maintain compatibility with existing usages?

Reply:
@cookbr great suggestion, are you able to amend your PR to support this?

Reply:
@cookbr I agree, we should try to maintain backwards compatibility because there are some customers still using the legacy Workflows. **Any change that you can update the PR?** Afterwards, I can review and merge it for you.